src/sentry/api/endpoints/relay/project_configs.py

```diff
@@ -26,6 +26,7 @@ def _sample_apm():
 class RelayProjectConfigsEndpoint(Endpoint):
     authentication_classes = (RelayAuthentication,)
     permission_classes = (RelayPermission,)
+    enforce_rate_limit = False
```

Comment:
Was this endpoint ever rate limited? Or was there some other logic in place that prevented rate limiting?

Reply:
In the past it was not, until general rate limits were introduced for the APIs. This change follows a discussion I've had with @AniketDas-Tekky last week, deciding that we'll disable all rate limits on endpoints that are called exclusively by processing Relays.

Reply:
thanks for catching this @jan-auer, I was aware that we were introducing general rate limits but didn't think about impact to relay apis